File: BraindoWeb/Braindo/Common/Patient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Braindo.Common
{
    public class Patient
    {
        private int ID;
        private String name;
        private String surname;
        private int age;
        private String career;
        private String state;
        private String municipality;
        private String parish;
        private float totalAnxiety;
        private DateTime initialDate;
        private DateTime endingDate;
        private float month;
        private float year;
        private int error;

        public Patient()
        {

        }

        public Patient(int _id)
        {
            this.ID = _id;
        }

        public Patient(String _state)
        {
            this.state = _state;
        }

        public Patient(String _name, String _surname)
        {
            this.name = _name;
            this.surname = _surname;
        }

        public Patient(int _id, String _name, String _surname)
        {
            this.ID = _id;
            this.name = _name;
            this.surname = _surname;
        }

        public Patient(int _ID, String _name, String _surname, int _age, String _career, String _state, String _municipality, String _parish)
        {
            this.ID = _ID;
            this.name = _name;
            this.surname = _surname;
            this.age = _age;
            this.career = _career;
            this.state = _state;
            this.municipality = _municipality;
            this.parish = _parish;
        }

        public Patient(String _name, String _surname, int _age, String _career, String _state, String _municipality, String _parish)
        {
            this.name = _name;
            this.surname = _surname;
            this.age = _age;
            this.career = _career;
            this.state = _state;
            this.municipality = _municipality;
            this.parish = _parish;

        }

        public Patient(int _age, String _career, String _state, float _totalAnxiety)
        {
            this.age = _age;
            this.career = _career;
            this.state = _state;
            this.totalAnxiety = _totalAnxiety;

        }

        public Patient(String _state, String _municipality, float _totalAnxiety)
        {
            this.state = _state;
            this.municipality = _municipality;
            this.totalAnxiety = _totalAnxiety;

        }

        public Patient(int _age, float _totalAnxiety)
        {
            this.age = _age;
            this.totalAnxiety = _totalAnxiety;

        }

        public Patient(String _career, float _totalAnxiety)
        {
            this.career = _career;
            this.totalAnxiety = _totalAnxiety;

        }

        public Patient(DateTime _initialDate, DateTime _endingDate)
        {
            this.initialDate = _initialDate;
            this.endingDate = _endingDate;
        }

        public Patient(float _month, float _year, float _totalAnxiety)
        {
            this.month = _month;
            this.year = _year;
            this.totalAnxiety = _totalAnxiety;
        }


        public int _ID
        {
            get { return ID; }
            set { ID = value; }
        }

        public String _Name
        {
            get { return name; }
            set { name = value; }
        }

        public String _Surname
        {
            get { return surname; }
            set { surname = value; }
        }

        public int _Age
        {
            get { return age; }
            set { age = value; }
        }

        public String _Career
        {
            get { return career; }
            set { career = value; }
        }

        public String _State
        {
            get { return state; }
            set { state = value; }
        }

        public String _Municipality
        {
            get { return municipality; }
            set { municipality = value; }
        }

        public String _Parish
        {
            get { return parish; }
            set { parish = value; }
        }

        public DateTime _InitialDate
        {
            get { return initialDate; }
            set { initialDate = value; }
        }

        public DateTime _EndingDate
        {
            get { return endingDate; }
            set { endingDate = value; }
        }

        public int _Error
        {
            get { return error; }
            set { error = value; }
        }

    }
}

File: Braindo/app/src/main/java/com/app/braindo/braindo/common/entities/Patient.java
package com.app.braindo.braindo.common.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Patient implements Serializable {
    @SerializedName("_id")
    private int _id;
    @SerializedName("_firstname")
    private String _firstname;
    @SerializedName("_lastname")
    private String _lastname;
    @SerializedName("_age")
    private int _age;
    @SerializedName("_career")
    private String _career;
    @SerializedName("_addressInt")
    private int _addressInt;
    @SerializedName("_state")
    private String _state;
    @SerializedName("_municipality")
    private String _municipality;
    @SerializedName("_parish")
    private String _parish;
    @SerializedName("_email")
    private String _email;
    @SerializedName("_error")
    private int _error;

    public Patient(){}

    public Patient(int id, String firstname, String lastname){
        _id = id;
        _firstname = firstname;
        _lastname = lastname;
    }

    public Patient(int id, String email){
        _id = id;
        _email = email;
    }
    public Patient(int id, String firstname, String lastname, int age, String career){
        _id = id;
        _firstname = firstname;
        _lastname = lastname;
        _age = age;
        _career = career;
    }

    public Patient(int id, String firstname, String lastname, int age, String career, int addressInt, String email){
        _id = id;
        _firstname = firstname;
        _lastname = lastname;
        _age = age;
        _career = career;
        _addressInt = addressInt;
        _email = email;
    }

    public Patient(int id, String firstname, String lastname, int age, String career, String state, String municipality, String parish, String email){
        _id = id;
        _firstname = firstname;
        _lastname = lastname;
        _age = age;
        _career = career;
        _state = state;
        _municipality = municipality;
        _parish = parish;
        _email = email;
    }


    public int get_id() {
        return _id;
    }

    public void set_id(int _id) {
        this._id = _id;
    }

    public String get_firstname() {
        return _firstname;
    }

    public void set_firstname(String _firstname) {
        this._firstname = _firstname;
    }

    public String get_lastname() {
        return _lastname;
    }

    public void set_lastname(String _lastname) {
        this._lastname = _lastname;
    }

    public int get_age() {
        return _age;
    }

    public void set_age(int _age) {
        this._age = _age;
    }

    public String get_career() {
        return _career;
    }

    public void set_career(String _career) {
        this._career = _career;
    }

    public int get_addressInt() {
        return _addressInt;
    }

    public void set_addressInt(int _addressInt) {
        this._addressInt = _addressInt;
    }

    public String get_state() {
        return _state;
    }

    public void set__state(String _state) {
        this._state = _state;
    }

    public String get_municipality() {
        return _municipality;
    }

    public void set_municipality(String _municipality) {
        this._municipality = _municipality;
    }

    public String get_parish() {
        return _parish;
    }

    public void set_parish(String _parish) {
        this._parish = _parish;
    }

    public String get_email() {
        return _email;
    }

    public void set_email(String _email) {
        this._email = _email;
    }

    public int get_error() {
        return _error;
    }

    public void set_error(int _error) {
        this._error = _error;
    }

}
